.github/workflows/clp-core-build.yaml

```
@@ -371,4 +373,4 @@
             .task/checksum/lint-check-cpp-static-full
             .task/checksum/utils-cpp-lint-clang-tidy-*
             build/lint-clang-tidy
-          key: "${{steps.cache-restore-lint-check-cpp-static-full.outputs.cache-primary-key}}"
+          key: "${{steps.cache-restore-lint-check-cpp-static-full.outputs.cache-primary-key}}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)